src/Main.cpp

```diff
@@ -556,3 +561,25 @@ int main(int argc, char *argv[])
 
     return build_result;
 }
+
+#if defined(TUNDRA_WIN32)
+// On Windows we want to parse the command line using wide characters for Unicode support
+int wmain(int argc, wchar_t* argv[], wchar_t* envp[])
+{
+    char** argv_mbcs = new char* [argc];
+    for (int i = 0; i < argc; i++)
+    {
+        std::string argv_mbcs_str = ToMultiByteUTF8String(argv[i]);
+        argv_mbcs[i] = new char[strlen(argv_mbcs_str.c_str()) + 1];
+        strcpy(argv_mbcs[i], argv_mbcs_str.c_str());
```

Comment:
It's probably not a performance issue, but could you avoid these string copies by creating an array of std::string's in the outer scope and doing `argv_mbcs[i] = strings[i].c_str();` ?